nereid/nereid/data/default_data/state/region/config.yml

```diff
@@ -90,18 +108,91 @@ api_recognize:
         validation_fallback: NTFacility
         tmnt_performance_facility_type: Sand Filter
 
+      swale: 
+        validator: FlowAndRetFacility
+        validation_fallback: NTFacility
+        tmnt_performance_facility_type: Vegetated Swale
+
+      hydrodynamic_separator: 
+        validator: FlowFacility
+        validation_fallback: NTFacility
+        tmnt_performance_facility_type: Hydrodynamic Separator
+
+      dry_well:
+        validator: DryWellFacility
+        validation_fallback: NTFacility
+        tmnt_performance_facility_type: ¯\_(ツ)_/¯
+      
+      cistern:
+        validator: CisternFacility
+        validation_fallback: NTFacility
+        tmnt_performance_facility_type: ¯\_(ツ)_/¯
+
+      dry_weather_diversion:
+        validator: LowFlowFacility
+        validation_fallback: NTFacility
+        tmnt_performance_facility_type: ¯\_(ツ)_/¯
+
 project_reference_data:
+  dry_weather_flow_table:
+    file: dry_weather_flow.json
+    seasons:
+      winter:
+        - oct
+        - nov
+        - dec
+        - jan
+        - feb
+        - mar
+      summer:
+        - apr
+        - may
+        - jun
+        - jul
+        - aug
+        - sep
+    
   met_table:
     file: met_data.json
+    volume_nomo:
+      x_col: size_fraction
+      t_col: ddt_hr
+      y_col: capture_fraction
+    flow_nomo:
+      x_col: intensity_inhr
+      t_col: tc_minutes
+      y_col: performance_frac
+
     preprocess:
       - remaps:
         - left: rain_gauge
-          right: nomograph_file
+          right: volume_nomograph
+          how: left  # interpreted as 'right' = mapping['left'] if 'left' exists
+          mapping:
+            100_LAGUNABEACH: nomographs/100_LAGUNABEACH_volume_nomo.csv
+            1130_LAGUNA_AUDUBON: nomographs/1130_LAGUNA_AUDUBON_volume_nomo.csv
+
+        - left: rain_gauge
+          right: flow_nomograph
           how: left  # interpreted as 'right' = mapping['left'] if 'left' exists
           mapping:
-            100_LAGUNABEACH: nomographs/clean_nomo.csv
-            1130_LAGUNA_AUDUBON: nomographs/clean_nomo.csv
+            100_LAGUNABEACH: nomographs/100_LAGUNABEACH_flow_nomo.csv
+            1130_LAGUNA_AUDUBON: nomographs/1130_LAGUNA_AUDUBON_flow_nomo.csv
   
+  dry_weather_tmnt_performance_table:
+    file: dry_weather_bmp_params.json
+    facility_column: facility_type
+    pollutant_column: pollutant
+    preprocess:
+      - remaps:
+        - left: unit
+          right: -- # with how = replace, the right key is not parsed at all.
+          how: replace  # interpreted as 'left' = mapping['left']
+          mapping:
+            # units must be compatible with the `pint` library
+            "MPN/100mL": "MPN/_100ml"
+            "MPN/100 mL": "MPN/_100ml"
```

Comment:
yes, case matters, but it's addressed in line 4 of the config.yml file which remaps MPN = count = mpn and _100ml = 100 * milliliter = _100mL. 

these `pint_unit_registry` mappings are loaded by the `update_unit_registry` decorator function that wraps any task that contains code that does a unit conversion.

also, we do not need to be more explicit or tolerant of case on lines 193-194 or 207-208 because the project config should actually know what the terms are in the project data files, that's why it exists.